tests/intrinsic/uart.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include <stdint.h>
+
+struct uartlite_regs {
+    volatile unsigned int rx_fifo;
+    volatile unsigned int tx_fifo;
+    volatile unsigned int status;
+    volatile unsigned int control;
+};
```

Comment:
Can we reuse `struct uartlite_regs` and the constants with `emulator/src/uartlite.h`?

Reply:
I will preserve this due to the uartlite driver is common in most system software including opensbi, u-boot, linux.